src/main/java/duke/task/ToDoTask.java

```diff
@@ -0,0 +1,18 @@
+package duke.task;
+
+/**
+ * Represents a To-Do Task.
+ */
+public class ToDoTask extends Task {
+
+    private static final String EVENT_SYMBOL = "T";
```

Comment:
I like how you created a constant for EVENT_SYMBOL instead of hard-coding into you methods.